pcsx2-gsrunner/comparer.js

```diff
@@ -0,0 +1,111 @@
+/* Worst script known to man */
+/* Sources:
+   https://www.w3schools.com/howto/howto_js_lightbox.asp
+   https://css-tricks.com/prevent-page-scrolling-when-a-modal-is-open/
+*/
+
+function openModal() {
+  document.body.style.position = 'fixed';
+  document.body.style.top = `-${window.scrollY}px`;
+  document.getElementById("myModal").style.display = "block";
+  document.getElementById("myModal").focus();
+  setImageIndex(0);
+}
+
+function closeModal() {
+    document.getElementById("myModal").style.display = "none";
+    const scrollY = document.body.style.top;
+    document.body.style.position = '';
+    document.body.style.top = '';
+    window.scrollTo(0, parseInt(scrollY || '0') * -1);
```

Comment:
Did you mean `scrollY | '0'` or maybe `+ '0'` as javascript uses for appending strings.

Reply:
Uhh, that's what I nabbed from the linked page, but I'm guessing it's to resolve it to 0 in case scrollY was undefined, otherwise you'd get nan iirc